doc/Rules.md

```diff
@@ -47,7 +47,7 @@ when skipped - especially for accessibility scenarios. More information:
 
 <a name="md002"></a>
 
-## ~~MD002 - First heading should be a top level heading~~
+## ~~MD002 - First heading should be a top-level heading~~
```

Comment:
I'm fine with this change, but please also make this update to the rule description in README.md and md002.js. You may also need to run the update schema script in package.json to update the schema file. (Global search is a good way to confirm.) Thanks!